test-new/e2e/GuardianVerifierBase.t.sol

```diff
@@ -427,6 +469,7 @@ abstract contract OwnableValidatorRecovery_AbstractedRecoveryModule_Base is
     ) public {
         IGuardianVerifier.ProofData
             memory proofData = getAcceptanceEmailProofData(
```

Comment:
Why do we call this here and also on line 494?

Reply:
Actually I just noticed, I don't need to do call that again, I was dealing guardianType case by case so slipped up.

Reply:
Could you remove the duplicate call in the next PR? Happy to merge this now as only a small comment

Reply:
Ahh! I did locally, forgot to push 🥲

will do in the next one